src/calk.ts

```diff
@@ -58,4 +84,19 @@ export default class Calculator {
     }
     return points;
   }
+
+  showFrame(numberFrame: number): string {
+    let str: string = `
+    ----------------
+    frame [${arrayFrames[numberFrame].numberFrame}]
+    result [${arrayFrames[numberFrame].result}] 
+    frameScore [${arrayFrames[numberFrame].frameScore}]
+    total [${arrayFrames[numberFrame].total}]
+    ----------------
+    `
+    
+    console.log(str)
```

Comment:
@Kod-dasm Get rid of console log. I would propose to split this method into multiple like: frameScore(frameNumber: number): number, frameThrows(frameNumber: number): [number, number?] - cannot ask for not started frames, totalScoreAtFrame(frameNumber: number) and update tests accordingly. Current method is hard to use and test. Let's imagine you need to use these 4 proposerties from showFrame which returns string. In that case you will need to parse it somehow which is not desirable when we talk about a class library.